drv/stm32h7-rng/src/main.rs

```diff
@@ -23,73 +23,134 @@ use userlib::*;
 
 task_slot!(SYS, sys);
 
+ringbuf::ringbuf!(Trace, 32, Trace::Blank);
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+enum Trace {
+    Blank,
```

Comment:
Nit: do you prefer `Blank` to `None`?  `Trace::None` is very commonly used as the default type for other ringbufs.

Reply:
I know, I really dislike this since None is imported in the prelude from Option. What I actually want is for ringbufs to be uninitialized until filled in so we can stop doing this, but for want of that...